docs/consensus/proof_of_archival_storage.md

```diff
@@ -140,7 +140,7 @@ This extra data is added to the end of the [SCALE-encoded](https://docs.substrat
     
     Determine which pieces are to be downloaded for this sector: 
     
-    1. Index the sectors sequentially and for each sector derive `sector_id = keyed_hash(public_key_hash, sector_index)`, where `sector_index` is the sector index in the plot.
+    1. Index the sectors sequentially and for each sector derive `sector_id = keyed_hash(public_key_hash, [sector_index, history_size])`, where `sector_index` is the sector index in the plot and `history_size` is the current history size at the time of sector creation.
```

Comment:
Here and below:
```suggestion
    1. Index the sectors sequentially and for each sector derive `sector_id = keyed_hash(public_key_hash, sector_index || history_size)`, where `sector_index` is the sector index in the plot and `history_size` is the current history size at the time of sector creation.
```

This is the concatenation syntax we seem to be using elsewhere.